src/shared/json.c

```diff
@@ -1066,6 +1066,42 @@ double json_variant_real(JsonVariant *v) {
         return 0.0;
 }
 
+int json_variant_bytes(JsonVariant *v, uint8_t **p, size_t *n) {
+        assert_return(p, -EINVAL);
+        assert_return(n, -EINVAL);
```

Comment:
Please use `assert()`, rather than `assert_return()`, which is for public functions.